public class Task {
    public static void main(String[] args){
        try{
            int data = 50/0;
            System.out.println("rest of the code");
        }
        catch(ArithmeticException e){
            System.out.println(e);
        }
        System.out.println("rest of the code");
    }
}
//examples of try catch finally
/*try
        {
        int arr[]= {1,3,5,7};
        System.out.println(arr[10]);
        }
        // handling the array exception
        catch(ArrayIndexOutOfBoundsException e)
        {
        System.out.println(e);
        }
        System.out.println("rest of the code");
        }

        } */
//try{
    //int data=50/5;
   // System.out.println(data);
}
//catch(Null pointer exception e){
   // System.out.println(e);
   // finally{
        //System.out.println("rest of the code");
       // }
       // System.out.println("final code execute");


    //



        }

